wdn/templates_4.1/less/layouts/share.less

```diff
@@ -183,6 +183,18 @@
     }
 }
 
+/* the share widget dropdown needs some focus love because it's icon is absolutely positioned */
+.wdn-dropdown-widget-label.focused .wdn-icon-share {
+	/* firefox */
+	outline: 1px dotted #212121;
+	/* -webkit-focus-ring-color = '#5B9DD9' */
```

Comment:
I meant explain what this line comment is for and why webkit hacks are necessary. Focus-like styles make perfect sense. 

But it you want to try for getting rid of the label/checkbox approach. I'm all for it. That approach was to support javascript disabled browsers. And that simply doesn't make sense for the social share that almost certainly requires JS.